### DISCUSSION TITLE
How to create multilevel parameter hierarchy?

### DISCUSSION THREAD

**Discussion by agent3133** [atoti/atoti#500] [Q&A] [answered]
Hi,

you have a handy method to create single level hierarchy for the array index:
`cube.create_parameter_hierarchy_from_members()`

is there any possibility to create multilevel hierarchies for array indexes? 

Best Regards

**Comment by bloussou** [comment, marked as answer]
Hello,
This is not supported at the moment. It has been requested in this ticket : https://github.com/atoti/atoti/issues/469
